test/subscription.test.js

```diff
@@ -60,4 +60,15 @@ describe('Subscription', () => {
       sub.emit('ready');
     }).not.toThrow(TypeError);
   });
+
+  it('should not trigger error if `whenReady` is called multiple times', () => {
+    const sub = new Subscription('foo', 'fooSub', {}, conn);
+    // Calling more than once to subscribe multiple times to the `ready` event
+    sub.whenReady();
+    sub.whenReady();
```

Comment:
Perhaps also assert that the promise is called for the last `whenReady()`?

Reply:
Agree, will do so!

Reply:
Done: https://github.com/mixmaxhq/publication-client/pull/40/commits/e1a99dbbe68b5310992c3d6572a6e0936e2741f7